data/kubernetes-operator-published-branches.yaml

```diff
@@ -13,13 +13,14 @@ version:
     - '1.8'
```

Comment:
I cant comment above this line, but this is for line 3 and line 10:

We need to add a `- 1.12` under both `published` and `active`. This is because the `master` branch acts as `1.12`, and it needs to be in these lists in order to show up in the drop-down.

Reply:
Done